python/tk_multi_launchapp/base_launcher.py

```diff
@@ -334,3 +345,32 @@ def launch_from_path_and_context(self, path, context, version=None):
         :param version: (Optional) Specific version of DCC to launch.
         """
         raise NotImplementedError
+
+    def _sort_versions(self, versions):
+        """
+        Uses standard python modules to determine how to sort arbitrary version numbers. A version
+        number consists of a series of numbers, separated by either periods or strings of letters.
+        When comparing version numbers, the numeric components will be compared numerically, and
+        the alphabetic components lexically. For example:
+
+            1.1 < 1.2 < 1.3
+            1.2 < 1.2a < 1.2ab < 1.2b
+
+        This methodology is also used in Desktop. The input list of versions is not modified.
+
+        :param list versions: List of version "numbers" (may be strings)
+        :returns: List of sorted versions
+        """
+        # Do not sort the incoming versions in place.
+        sort_versions = [version for version in versions]
+
+        def version_cmp(left_version, right_version):
+            if util.is_version_newer(left_version, right_version):
```

Comment:
see comment above - this can probably be simplified by using `LooseVersion` directly